services/provider/server/storageclaim.go

```diff
@@ -143,25 +143,25 @@ func (s *storageClassClaimManager) Delete(ctx context.Context, consumerUUID, sto
 	}
 	if err := s.client.Delete(ctx, storageClassClaimObj, &deleteOption); err != nil {
 		if kerrors.IsNotFound(err) {
-			klog.Warningf("StorageClassClaim %q not found for consumer %q and claim %q", generatedClaimName, consumerUUID, storageClassClaimName)
+			klog.Warningf("StorageClassRequest %q not found for consumer %q and claim %q", generatedClaimName, consumerUUID, storageClassClaimName)
 			return nil
 		}
-		return fmt.Errorf("failed to delete StorageClassClaim %q for consumer %q and claim %q. %v", generatedClaimName, consumerUUID, storageClassClaimName, err)
+		return fmt.Errorf("failed to delete StorageClassRequest %q for consumer %q and claim %q. %v", generatedClaimName, consumerUUID, storageClassClaimName, err)
 	}
 
-	klog.Infof("successfully deleted StorageClassClaim %q for consumer %q and claim %q", generatedClaimName, consumerUUID, storageClassClaimName)
+	klog.Infof("successfully deleted StorageClassRequest %q for consumer %q and claim %q", generatedClaimName, consumerUUID, storageClassClaimName)
 
 	return nil
 }
 
-// Get returns the storageClassClaim resource using storageClassClaimName
+// Get returns the StorageClassRequest resource using storageClassClaimName
 // and consumerUUID.
-func (s *storageClassClaimManager) Get(ctx context.Context, consumerUUID, storageClassClaimName string) (*ocsv1alpha1.StorageClassClaim, error) {
-	generatedClaimName := getStorageClassClaimName(consumerUUID, storageClassClaimName)
-	storageClassClaimObj := &ocsv1alpha1.StorageClassClaim{}
+func (s *storageClassRequestManager) Get(ctx context.Context, consumerUUID, storageClassClaimName string) (*ocsv1alpha1.StorageClassRequest, error) {
```

Comment:
storageclassclaimName need to be replaced with storageRequestName in all the places in the file.